WireDomain/Sources/WireDomain/UseCases/IndividualToTeamMigrationUseCaseImplementation.swift

```diff
@@ -46,4 +55,34 @@ public struct IndividualToTeamMigrationUseCaseImplementation: IndividualToTeamMi
             }
         }
     }
+
+    private func createTeamLocally(
+        id: UUID,
+        name: String
+    ) async throws {
+        try await context.perform { [context] in
+            let team = Team.fetchOrCreate(
+                with: id,
+                in: context
+            )
+
+            team.name = name
+            team.needsToBeUpdatedFromBackend = true
+
+            // Probably there's nothing to fetch since it's a brand
+            // new team, but just in case.
+            team.needsToDownloadRoles = true
+            team.needsToRedownloadMembers = true
```

Comment:
we create the team locally do we need to do: `team.needsToUpdateFromBackend = true` @johnxnguyen ?